docker-compose.override.yaml

```
@@ -50,7 +50,7 @@ services:
       - ./dev/.env.docker-compose
     environment:
       RDWATCH_POSTGRESQL_URI: "postgresql://rdwatch:secretkey@postgresql:5432/rdwatch"
-      RDWATCH_REDIS_URI: "redis://:secretkey@redis-master:6379"
+      RDWATCH_REDIS_URI: "redis://:redis:6379"
```

Celery Worker wouldn't connect in docker dev mode
```suggestion
      RDWATCH_REDIS_URI: "redis://redis:6379"
```